src/main/java/com/sfuentes/ClubMember.java

```diff
@@ -0,0 +1,40 @@
+package com.sfuentes;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ClubMember {
+  int memberID;
+  Map<Integer, List<Double>> weightPerScale;
+  double previousWeight;
+
+  public ClubMember(int memberID, double previousWeight, Map<Integer, List<Double>> weightPerScale) {
```

Comment:
What is the purpose of memberId? I see in your code is always 1, but not sure how it works.